Jenkinsfile

```diff
@@ -17,27 +17,35 @@ pipeline {
 	stages {
 		stage('Install dependencies') {
 			steps {
-				sh '''
-				cp -r ~/cache/${CHANGE_TARGET:-${BRANCH_NAME:-development}}/node_modules ./ || true
```

Comment:
Following up the template mainly.

Reply:
I think we had it for a reason @fchavant - can you have a look please? :)

Reply:
I think main reason is the `cypress` dependency, which takes quite long time.
However, I seems like not being a problem with current `1.0.0`, also test should be as stateless as possible.

Reply:
Okay, that would be quite cool actually. Yes, I agree to your point.

Reply:
@Tosch110 I seem to remember it was to prevent downloading cypress every single time, as @shuse2 pointed out.

Reply:
@fchavant 
Should we keep the cache for the jenkins work load?

Reply:
@shuse2 from your comment I assume cypress was gone from 1.0.0 -- in that case we do not need any cache.

Reply:
Cypress is still around.

Reply:
@fchavant Ah, sorry. What I meant was not being problem with this *current* branch.
Install dependency step finishes in 1.5 min

Reply:
I think probably 1 minute of the 1.5 minutes is alone cypress. Since we do not care so much about cypress version, maintaining the cache here would make sense to me.

Reply:
If it is not critical to have cache here, I prefer not having the cache here.
For example, the version of cypress was affecting this PR, which would not be able to be detected easily if it was cached.

Reply:
Oh. I was not aware this was causing problems already. In that case not cached is fine with me :+1: